WORKFLOW.md

```diff
@@ -38,10 +38,24 @@ out-of-cycle release to the production environment. This may include
 non-critical bug fixes, documentation updates, CI/CD changes, dependency 
 updates, or other tooling changes.
 
-Feature development begins by creating a new branch off of `main`:
+Feature development begins by creating a new branch off of `main`.
+
+> NOTE: **Shortcut integration**
+>
+> It is recommended that you include the Shortcut story number in the branch name.
```

Comment:
I'm actually a bit on the fence here. As this is an open source repo, I 1) Don't think automation is setup and 2) I'm not sure we want to expose the Shortcut numbers etc. as that provides no useful info to anyone outside the org looking at this repo.

Reply:
I think including the story number in the branch name doesn't cost much but I'm fine removing this additional information here.

I do think we should automate the Shortcut integration, though.